### ISSUE TITLE
Please note versions in Bazel docs

### ISSUE THREAD

**Issue by jart** [bazelbuild/bazel#1788]
On TensorFlow and Closure Rules, I want to be able to provide at least some level of support for old versions of Bazel.

This is currently not easy to do, because the documentation doesn't tell me at which version a feature was introduced, or became deprecated.

For example, [ConfigurationTransition](https://www.bazel.io/versions/master/docs/skylark/lib/ConfigurationTransition.html) doesn't say at which version `"host"` instead of `HOST_CFG` started working.

Or the documentation for [alias()](https://www.bazel.io/versions/master/docs/be/general.html#alias) doesn't tell me in which version it was introduced.

I understand Bazel isn't quite at 1.0 yet. But I still think taking the extra time to do this would be worthwhile.

**Comment by damienmg** [comment]
Very good idea, I don't know how much can be automated. +David is working
on versioning our documentation with our releases.

**Comment by pcj** [comment]
I second that.  When rules_go made the change from HOST_CFG to "host" in https://github.com/bazelbuild/rules_go/pull/100 it transitively broke support for bazel 0.3.0.

**Comment by davidzchen** [comment]
Agreed. Once this is done, the documentation pages will provide both the version and Git commit hash. I plan to have this done within the next week.

**Comment by davidzchen** [comment]
FYI since we have finished with the Bazel website restructuring, I am working on completing the tooling for versioning the documentation at docs.bazel.build. I'm de-duping this against #579 since the the change I am working on covers both.

Duplicate of #579
